packages/int2lm/package.py

```diff
@@ -14,18 +14,18 @@ class Int2lm(MakefilePackage):
     and/or boundary data for the COSMO-Model."""
 
     homepage = "http://www.cosmo-model.org/content/model/"
-    url      = "https://github.com/MeteoSwiss-APN/int2lm/archive/v2.8.3.tar.gz"
+    url      = "https://github.com/MeteoSwiss-APN/int2lm/archive/refs/tags/v2.8.4.tar.gz"
```

Comment:
for info I don't think you need to change this url, I don't remember for what it is needed @elsagermann - can you refresh me.